tools/team_list.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import requests
+
+
+project = 'scikit-image'
+team = 'core'
+emeritus_team = 'emeritus'
+team_url = f'https://api.github.com/orgs/{project}/teams/{team}/members'
+emeritus_team_url = f'https://api.github.com/orgs/{project}/teams/{emeritus_team}/members'
+
+
+token = os.environ.get('GH_TOKEN', None)
+if token is None:
+    print("No token found.  Please export a GH_TOKEN with permissions "
+          "to read team members.")
+    sys.exit(-1)
+
+
+def api(url):
+    return requests.get(url=url,
+                        headers={'Authorization': f'token {token}'}).json()
+
+
+resp = api(team_url)
+team = sorted(resp, key=lambda user: user['login'].lower())
+
+resp = api(emeritus_team_url)
+emeritus_team = sorted(resp, key=lambda user: user['login'].lower())
+
+
+def render_team(team):
+    for member in team:
+        profile = api(member['url'])
+
+        print(f"""
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="{member['avatar_url']}&s=40"/>
```

Comment:
```suggestion
       <img src="{member['avatar_url']}&s=40" alt="Avatar picture of {profile['name']}."/>
```

Reply:
Also adding a `loading="lazy"` here might be worth it for reducing requests to third parties before rendering the start page?